_commands/tools/make.md

```diff
@@ -2,8 +2,115 @@
 ---
 
 make
-----
+-------
 
-<em>TODO: Placeholder</em>
+Simply put, this command can compile code into an executable without having to type a super long compilation sequence. We can use the make utility to automate many tasks in the development process, including compilation and regression testing.
 
+~~~ bash
+$ make main
+c++     main.cpp   -o main
+$ ./main
+~~~
 
+
+<!--more-->
+
+### Useful Options / Examples
+
+In order to do more, we need to create something called a Makefile. There's quite a few tutorials online for how to write a Makefile, but it's usually easiest to grab a Makefile that already exists and modifying the recipes for your project. Take a look at the Makefile below from EECS 280.
+
+Makefiles are comprised of "targets" which execute some task.
+
+~~~ bash
```

Comment:
Does kramdown have a syntax highlighter for `make` instead of `bash` (think it does), would be better here